src/frontend-samples/clashdetection-sample/ClashDetectionJsonData.ts

```diff
@@ -0,0 +1,744 @@
+// Clash Detection Result Sample Data
```

Comment:
We can remove this file now that we're using the API right?

Reply:
I left it there as a sample (as well as a backup in case the API calls fail).  I have a link to it in the sample readme.